docs/application/native/guides/machine-learning/machine-learning-inference.md

```diff
@@ -298,6 +298,135 @@ Followings are the available elements:
     char pipeline[] = "videotestsrc is-live=true ! videoconvert ! tensor_converter ! output-selector name=outs outs.src_0 ! tensor_sink name=sink0 async=false outs.src_1 ! tensor_sink name=sink1 async=false"
     ```
 
+- **If**
+
+    The if element (tensor_if) controls its src-pad based on the values (other/tensor(s)) of its sink-pad.
+
+    For example, you may skip frames if there is no object detected with high confidence.
```

Comment:
Question: 
- From Line 303-307, can't we have the text in paragraph form? Is the text intentionally formatted with line breaks?

The sentences 1 and 2 seem to be disjoint.

1. The **if** (tensor_if) element controls its src-pad based on the values of other tensors of its sink-pad. 

2. For example, you may skip frames if there is no object detected with high confidence.

3. The input and output stream data type is either `other/tensor` or `other/tensors`.